conf.pri.windows

```diff
@@ -0,0 +1,58 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost)
+INCLUDEPATH += $$quote(C:/qBittorrent/libtorrent/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/zlib/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
+# Point these to the lib folders
+LIBS += $$quote(-LC:/qBittorrent/boost/stage/lib)
+LIBS += $$quote(-LC:/qBittorrent/libtorrent/lib)
+LIBS += $$quote(-LC:/qBittorrent/zlib/lib)
+LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
+
+# Adapt the lib names/versions accordingly
+# If you want to use Boost auto-linking then disable
+# BOOST_ALL_NO_LIB below and omit Boost libraries here
+CONFIG(debug, debug|release) {
+    LIBS += libtorrentd.lib \
+            libboost_system-vc140-mt-sgd-1_64.lib
+}
+else {
+    LIBS += libtorrent.lib \
+            libboost_system-vc140-mt-s-1_64.lib
+}
+LIBS += libeay32.lib ssleay32.lib
+LIBS += zlib.lib
+# ...or if you use MinGW
```

Comment:
why don't you use `win32-g++* {} else {}` here?

Reply:
or both `win32-msvc*`, `win32-g++*`...

Reply:
This is just example config. User will edit it and remove unneeded lines.

Reply:
It's clearer for me to have `win32-<compiler> {}` block, so I know immediately where I need to modify, just saying.
(not a change request)

Reply:
Really I would have only one example branch.